packages/rrweb/src/replay/index.ts

```diff
@@ -1798,18 +1798,24 @@ export class Replayer {
                   const newSn = mirror.getMeta(
                     target as Node & RRNode,
                   ) as serializedElementNodeWithId;
-                  Object.assign(
-                    newSn.attributes,
-                    mutation.attributes as attributes,
```

Comment:
this used to be a mutating change to the attributes reference, passed to buildNodeWithSN and causing the logic to not go as intended down the path